README.md

```diff
@@ -113,7 +113,7 @@ The storefront will be accessible at
 [http://localhost:3000/](http://localhost:3000/) and the admin can be found at
 [http://localhost:3000/admin/](http://localhost:3000/admin/).
 
-For information on how to customize your store, check out the [customization guides](https://guides.solidus.io/developers/customizations/overview.html).
+For information on how to customize your store, check out the [customization guides](https://guides.solidus.io/customization/customizing-your-storefront).
```

Comment:
Hey @pokonski, thanks for catching this!

I noticed there's also this link that unfortunately leads to a 404:

https://github.com/solidusio/solidus/blob/51f44e13c0ee967645f54a4a074ea262988c8a40/bin/sandbox#L104C4-L104C80
```
🚀 https://guides.solidus.io/developers/getting-started/first-time-installation
```

instead of 
```
🚀 https://guides.solidus.io/getting-started/installing-solidus
```
If you could add this fix as well, that'd be awesome!

Reply:
It probably makes sense to also add those to the guides redirects https://github.com/solidusio/edgeguides/blob/9bf18e4910f5124075dcd3041ce7368714f879ab/docusaurus.config.js#L178-L186

Reply:
Thank you, @elia! Here's the PR link: https://github.com/solidusio/edgeguides/pull/126